api/user.go

```diff
@@ -1355,6 +1365,13 @@ func updateUser(c *Context, w http.ResponseWriter, r *http.Request) {
 			go sendEmailChangeUsername(c, rusers[1].Username, rusers[0].Username, rusers[0].Email, c.GetSiteURL())
 		}
 
+		updatedUser := rusers[0]
+		updatedUser.Sanitize(map[string]bool{})
```

Comment:
cool I'll write a helper for that and update everywhere I see that code.

Thanks for the suggestion